pkg/apis/kops/v1alpha1/cluster.go

```diff
@@ -312,11 +320,6 @@ type DNSAccessSpec struct {
 // LoadBalancerType string describes LoadBalancer types (public, internal)
 type LoadBalancerType string
 
-const (
-	LoadBalancerTypePublic   LoadBalancerType = "Public"
-	LoadBalancerTypeInternal LoadBalancerType = "Internal"
-)
-
```

Comment:
II'd prefer to leave these where they were?  They're supposed to be close to where they are used.

Otherwise looks good - thanks!

Reply:
Ok, hold on

Reply:
thanks, has been updated.